source/mainnet/net/release-notes/release-notes-lp.rst

```diff
@@ -167,12 +167,16 @@ Wallets
 |bw|
 -------------------------
 
-    February 5, 2024
+    February XX, 2024
```

Comment:
Remember to update the date before publishing.

Reply:
Could you please update it to today?